scrapers/ESP/index.js

```diff
@@ -1,38 +1,135 @@
+import { LocalDate, DateTimeFormatter } from '@js-joda/core';
 import * as fetch from '../../lib/fetch.js';
-import * as parse from '../../lib/parse.js';
 import * as transform from '../../lib/transform.js';
-import * as rules from '../../lib/rules.js';
-
-// Set county to this if you only have state data, but this isn't the entire state
-// const UNASSIGNED = '(unassigned)';
+import * as parse from '../../lib/parse.js';
 
 const scraper = {
   country: 'ESP',
-  url: 'https://opendata.arcgis.com/datasets/48fac2d7de0f43f9af938852e3748845_0.csv',
+  url: 'https://github.com/datadista/datasets/tree/master/COVID%2019',
```

Comment:
Is this a primary source?

Reply:
It's as close as we can get: Official data is published in PDF form, this repo just republishes it in CSV.